charts/lakefs/templates/_fluffy.tpl

```diff
@@ -0,0 +1,94 @@
+{{/*
+fluffy resource full name
+*/}}
+{{- define "fluffy.fullname" -}}
+{{- $name := include "lakefs.fullname" }}
+{{- printf "%s-fluffy" $name  | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "fluffy.labels" -}}
+helm.sh/chart: {{ include "fluffy.chart" . }}
+{{ include "fluffy.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "fluffy.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "fluffy.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "fluffy.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "fluffy.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+
+{{/*
+Fluffy environment variables
+*/}}
+
+{{- define "fluffy.env" -}}
+env:
+  {{- if and .Values.fluffy.sso.enabled (.Values.fluffy.sso.oidc).client_secret }}
+  - name: FLUFFY_AUTH_OIDC_CLIENT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: {{ include "fluffy.fullname" . }}
+        key: oidc_client_secret
+  {{- end }}
+  {{- if and .Values.secrets (.Values.secrets).authEncryptSecretKey }}
+  - name: FLUFFY_AUTH_ENCRYPT_SECRET_KEY
+    valueFrom:
+      secretKeyRef:
+        name: {{ include "fluffy.fullname" . }}
+        key: auth_encrypt_secret_key
```

Comment:
Can we share the secret with lakeFS if we manage single chart and would like to have the same secret?